NuGet.Docs/Release-Notes/NuGet-3.2-RC.md

```diff
@@ -0,0 +1,99 @@
+
+# NuGet 3.2 RC Release Notes
+
+[NuGet 3.1.1 Release Notes](nuget-3.1.1)
+
+NuGet 3.2 release candidate was released September 2, 2015 as a collection of improvements and fixes for the 3.1.1 release.  Also, these are the first releases that are published first to the new dist.nuget.org repository.
+
+## New Features
+
+* Projects that live in the same folder can now have different project.json files in that folder specific to each project.  For each project, name the project.json file {ProjectName}.project.json and NuGet will properly reference and use that content for each project appropriately. ([1102](https://github.com/NuGet/Home/issues/1102)) 
```

Comment:
this is depending one a feature from a visual studio update to be released with win 10 tools